Dockerfile

```diff
@@ -1,8 +1,3 @@
-FROM node:12.18.3-alpine AS base
```

Comment:
why removing cdk cli ?

Reply:
We use the one in cloudfront-dist directly to avoid having to set the version in two different file.